docs/build/inferencing.md

```diff
@@ -107,6 +107,61 @@ The last command will generate a fat-binary for both CPU architectures.
 
 Note: unit tests will be skipped due to the incompatible CPU instruction set when doing cross-compiling.
 
+### AIX
```

Comment:
Change to `#### AIX` to align with the above `#### macOS`.

Reply:
Updated AIX section start to #### AIX